src/stories/Alert.stories.js

```diff
@@ -10,18 +10,30 @@ export const basicAlert = () => ({
   components: { BasicAlert },
   template: `
   <div>
-  <div style='margin-bottom:15px'>
-  <basic-alert title="success alert" type="success"></basic-alert> 
-  </div>
-  <div style='margin-bottom:15px'>
-  <basic-alert title="info alert" type="info"></basic-alert> 
-  </div>
-  <div style='margin-bottom:15px'>
-  <basic-alert title="warning alert" type="warning"></basic-alert> 
-  </div>
-  <div style='margin-bottom:15px'>
-  <basic-alert title="error alert" type="error"></basic-alert> 
-  </div>
+    <div style='margin-bottom:15px'>
+      <basic-alert 
+      title="success alert" 
+      type="success">
+      </basic-alert> 
```

Comment:
I think we can go for <basic-alert />

Reply:
?

Reply:
Ohh.. Didn't complete...😃  Not urgent but good to have single tag like <basic-alert /> for all component